client/packages/core/src/createRouteHandler.ts

```diff
@@ -0,0 +1,51 @@
+import { init } from './index.ts';
+
+export const createInstantRouteHandler = (config: {
+  appId: string;
+  apiURI?: string;
+}) => {
+  const admin = init({
+    appId: config.appId,
+    apiURI: config.apiURI,
+  });
+
+  async function handleUserSync(req: Request) {
+    const body = await req.json();
+    if (body.user && body.user.refresh_token) {
+      return new Response('sync', {
+        headers: {
+          // 24 hour expiry
+          'Set-Cookie': `instant_refresh_token=${body.user.refresh_token}; Path=/; HttpOnly; Secure; SameSite=Strict; Max-Age=86400`,
```

Comment:
What happens if you're testing on localhost? I would expect the `Secure` field to prevent this from working.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Set-Cookie#secure
On localhost addresses, browsers will still send secure cookies according to mdn. Maybe we want to give users more control over cookies anyways though